docs/manuals/man1/pki.1.md

```diff
@@ -0,0 +1,350 @@
+pki 1 "February 1, 2019" PKI "pki CLI"
+======================================
+
+NAME
+----
+
+pki - Command-Line Interface for accessing PKI services.
+
+SYNOPSIS
+--------
+
+`pki` [CLI options] &lt;command&gt; [command arguments]
+
+DESCRIPTION
+-----------
+
+The `pki` command provides a command-line interface allowing clients
+to access various services on the PKI server.
+These services include certificates, groups, keys, security domains, and users.
+  
+CLI OPTIONS
+-----------
+
+`-c` &lt;NSS database password&gt;  
```

Comment:
It'd be nice to do something about this too. I'm not sure if you want to do:

     `-c <NSS database password>` 

(and put it all in a single back-tick wrap), but I'm not sure how to indent the next part without turning it into a code block (to remove the `&nbsp;`). 

If you're willing to make it a list, two options would be doing something like this (I'm not sure how it shows up in the rendered man page though):

```markdown
  - `-c <NSS database password>`:  Specifies the NSS database
    password (mutually exclusive to the '-C' option).
```

```markdown
  - `-c <NSS database password>`
    Specifies the NSS database password (mutually exclusive to the '-C' option).
```

You can then remove all the `&nsbp;` stuff.  (By indenting after the newline in a bullet, the result will be indented as a multi-line bullet).

Reply:
I fixed the option and argument to use bold and italics, but I don't have a solution for the indentation that looks good online, offline, and in raw. I think we should prioritize the online and offline output first, but we certainly can revisit this once we figure out the best compromise.

Reply:
One possibility is to create a program to insert the `&nbsp;` before calling go-md2man.

Reply:
Why not just use four spaces (no `&nbsp;`)? That gets rendered fine in `man` (or at least, achieves the extra indent you want there. In the text file, your line doesn't become too long (as the text is on a different line than the options). And when viewing it on the web, it shows up on the same line, but that's fine since options are bold and arguments are italic, so you can tell when the parameters stop / the description begins?

https://gist.github.com/cipherboy/da84d2cb08a4a255aae5968969a28fe9

Reply:
OK, I think that's a decent compromise. Please take a look at the updated patch. The option description will appear in a separate line online because we have a line break with two spaces at the end of the previous line.